src/components/ConfirmedRoute.tsx

```diff
@@ -69,7 +59,7 @@ function ConfirmedRoute({mapboxAccessToken, transaction}) {
 
                     return {
                         id: `${waypoint.lng},${waypoint.lat},${index}`,
-                        coordinate: [waypoint.lng, waypoint.lat],
+                        coordinate: [waypoint.lng, waypoint.lat] as [number, number],
```

Comment:
Should be sufficient:
```suggestion
                        coordinate: [waypoint.lng, waypoint.lat] as const,
```